client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -26,14 +26,29 @@ const SubscriberListContainer = ( {
 	onClickView,
 	onClickUnsubscribe,
 }: SubscriberListContainerProps ) => {
-	const { grandTotal, total, perPage, page, pageChangeCallback, searchTerm, isLoading } =
-		useSubscribersPage();
+	const {
+		grandTotal,
+		total,
+		perPage,
+		page,
+		pageChangeCallback,
+		searchTerm,
+		isLoading,
+		subscribers,
+	} = useSubscribersPage();
 	useRecordSearch();
 
 	const isSimple = useSelector( isSimpleSite );
 	const isAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
 	const EmptyComponent = isSimple || isAtomic ? SubscriberLaunchpad : EmptyListView;
 
+	useEffect( () => {
+		if ( ! isLoading && subscribers.length === 0 && page > 1 ) {
+			const newPageNumber = page - 1;
```

Comment:
@JuanLucha The `useSubscribersQuery` returns the total number of pages (`pages` prop); we could pass these values to the context and use it here instead of decrementing the current page. It would avoid multiple useEffects in sequence.

Reply:
Good point! I'll do it! Thanks, @phcp !